etl/monitoring.py

```
@@ -342,45 +342,38 @@ def _check_system_time(self) -> HealthCheck:
+                "percent": usage_percent,
+                "available_gb": memory.available / (1024**3),
+                "total_gb": memory.total / (1024**3)
+            }
+        )
     
     def _check_disk_space(self) -> HealthCheck:
         """Check disk space."""
         try:
             import shutil
-            total, used, free = shutil.disk_usage("/")
+            import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)